docs/workload-identity-static-pv-mount.md

```diff
@@ -0,0 +1,196 @@
+# Example of static PV mount with workload identity
+
+> Note: 
+> - Available kubernetes version >= v1.20
+
+## prerequisite
+
+### 1. Create a cluster with oidc-issuer enabled and get the credential
+```
+export RESOURCE_GROUP=<your resource group name>
+export CLUSTER_NAME=<your cluster name>
+export REGION=<your region>
+
+az group create --name $RESOURCE_GROUP --location $REGION
+
+az aks create -n $CLUSTER_NAME -g $RESOURCE_GROUP --enable-oidc-issuer
```

Comment:
let's only mention `--enable-oidc-issuer` should be specified for cluster creation.